src/main/scala/edu/ie3/simona/model/participant2/WecModel.scala

```diff
@@ -67,24 +66,28 @@ class WecModel private (
     with ParticipantSimpleFlexibility[WecState]
     with LazyLogging {
 
-  override val initialState: ModelInput => WecState = { input =>
-    val weatherData = getWeatherData(input.receivedData)
+  override val initialState: (Long, ZonedDateTime) => WecState = { (tick, _) =>
     WecState(
-      input.currentTick,
-      weatherData.windVel,
-      weatherData.temp,
+      tick,
+      MetersPerSecond(0d),
+      Celsius(0d),
```

Comment:
This could be added to the default quantities.

Reply:
It could, but we're currently only using it one time, so I think it's not that useful

Reply:
Then we should keep this way.